lib/public_key/test/public_key_SUITE_data/rsa_md2.pem

```diff
@@ -0,0 +1,31 @@
+// %CopyrightBegin%
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
```suggestion
//
//
```

There needs to be another new line here... The license validation script was just merged, so at times it is a bit too picky... finding the right balance is hard.